metrics/prometheus.go

```diff
@@ -136,6 +137,10 @@ func parseKubectlOutput(output string) []Result {
 		}
 
 		os := strings.Split(o[0], "/")
+		if len(os) != 2 {
+			log.Logger.Error("Expected format: <resource-type>/<name>", "value", o, "full output", output)
+		}
```

Comment:
Perhaps there should be a continue here? 

There are cases where the output can have lines that don't match the pattern but I wouldn't necessarily consider the output as a whole invalid. In those cases I would expect the parser to ignore the line and continue, rather than allowing the program to barrel on into a panic.

See the `Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply` line here:
```
2019-12-16T14:44:46.956Z [ERROR] kube-applier: Expected format: <resource-type>/<name>: value=[Warning:, kubectl, apply, should, be, used, on, resource, created, by, either, kubectl, create, --save-config, or, kubectl, apply] full output="namespace/sys-auth configured (server dry run)
serviceaccount/ssh-key-manager unchanged (server dry run)
role.rbac.authorization.k8s.io/vault-configmap-applier unchanged (server dry run)
rolebinding.rbac.authorization.k8s.io/kube-applier unchanged (server dry run)
rolebinding.rbac.authorization.k8s.io/vault-configmap-applier unchanged (server dry run)
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
configmap/vault-tls configured (server dry run)
secret/k8s-auth-conf unchanged (server dry run)
secret/ssh-key-manager configured (server dry run)
service/k8s-auth-config unchanged (server dry run)
service/ssh-key-manager unchanged (server dry run)
limitrange/cpu-limit-range configured (server dry run)
deployment.apps/k8s-auth-config unchanged (server dry run)
deployment.apps/ssh-key-manager unchanged (server dry run)
networkpolicy.networking.k8s.io/default unchanged (server dry run)
ingress.networking.k8s.io/k8s-auth-config unchanged (server dry run)
ingress.networking.k8s.io/ssh-key-manager unchanged (server dry run)
"
panic: runtime error: index out of range [1] with length 1

goroutine 23 [running]:
github.com/utilitywarehouse/kube-applier/metrics.parseKubectlOutput(0xc0000c6900, 0x447, 0xa176a0, 0xc00023b540, 0xc000248418)
        /go/src/github.com/utilitywarehouse/kube-applier/metrics/prometheus.go:146 +0x4d4
github.com/utilitywarehouse/kube-applier/metrics.(*Prometheus).UpdateResultSummary(0xc00000c120, 0xc00021c060)
        /go/src/github.com/utilitywarehouse/kube-applier/metrics/prometheus.go:112 +0x372
github.com/utilitywarehouse/kube-applier/run.(*Runner).run(0xc000108120, 0xc00003dfaf, 0x1, 0x0)
        /go/src/github.com/utilitywarehouse/kube-applier/run/runner.go:75 +0x659
github.com/utilitywarehouse/kube-applier/run.(*Runner).Start(0xc000108120)
        /go/src/github.com/utilitywarehouse/kube-applier/run/runner.go:31 +0x87
created by main.main
        /go/src/github.com/utilitywarehouse/kube-applier/main.go:184 +0x72e
make: *** [run] Error 2
```

Reply:
Also the log contains the split line:
```
value=[Warning:, kubectl, apply, should, be, used, on, resource, created, by, either, kubectl, create, --save-config, or, kubectl, apply]
```

It would look a bit prettier if you logged the `line` variable instead.